### PR TITLE
Blog base

### PR DESCRIPTION
**Pull request by agusaldasoro**
# Summary
* Created a new blog using Postgres
* Setted the Application Home Page
* Created the Article Model
  * You can create a new article
  * All articles are shown

### TODO: 
Use _Slim_ instead of _.erb_

# Test Plan

# Trello Card
https://trello.com/c/NEuhvzR7

# Screenshots

### CODE REVIEW

File: blog/app/assets/javascripts/articles.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (JesusHerrera) [inline blog/app/assets/javascripts/articles.coffee]: Delete unused file

File: blog/app/assets/javascripts/welcome.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (JesusHerrera) [inline blog/app/assets/javascripts/welcome.coffee]: Delete unused file

File: blog/app/assets/stylesheets/articles.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the articles controller here.
```
Comment (JesusHerrera) [inline blog/app/assets/stylesheets/articles.scss]: Delete unused file

File: blog/app/assets/stylesheets/welcome.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the welcome controller here.
```
Comment (JesusHerrera) [inline blog/app/assets/stylesheets/welcome.scss]: Delete unused file

File: blog/app/controllers/articles_controller.rb [hunk 5]
```diff
@@ -0,0 +1,35 @@
+class ArticlesController < ApplicationController
```
Comment (JesusHerrera) [inline blog/app/controllers/articles_controller.rb]: Delete commented code

File: blog/app/controllers/articles_controller.rb [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+	# def index
+	# end
+
+	def show
+		@article = Article.find(params[:id])
```
Comment (JesusHerrera) [inline blog/app/controllers/articles_controller.rb]: Use 2 spaces for identation

File: blog/app/helpers/articles_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module ArticlesHelper
```
Comment (JesusHerrera) [inline blog/app/helpers/articles_helper.rb]: Delete unused file

File: blog/app/helpers/welcome_helper.rb [hunk 8]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
```
Comment (JesusHerrera) [inline blog/app/helpers/welcome_helper.rb]: Delete unused file

File: blog/test/controllers/articles_controller_test.rb [hunk 9]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (JesusHerrera) [inline blog/test/controllers/articles_controller_test.rb]: Delete unused file

**Comment by agusaldasoro** [comment]
Pull Request Fixed

**Pull request merged**